developer-docs-site/docs/concepts/delegated-staking.md

```diff
@@ -1,19 +1,20 @@
 ---
 title: "Delegated Staking"
-slug: " Delegated staking"
----
+slug: "delegated-staking"
```

Comment:
changing a slug, changes the url

Reply:
yeah i'm trying to fix the url because there's a space